client/src/components/CreateAuction.jsx

```diff
@@ -23,12 +25,13 @@ class CreateAuction extends React.Component {
     this.handleInputChange = this.handleInputChange.bind(this);
     this.handleCategoryChange = this.handleCategoryChange.bind(this);
     this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleImageInput = this.handleImageInput.bind(this);
+    this.handleImageCreate = this.handleImageCreate.bind(this);
   }
 
   handleInputChange(e) {
     const name = e.target.name;
     const value = e.target.value;
-    // console.log(`target name: ${name}\ntarget value ${value}`);
     this.setState({
       [name]: value
```

Comment:
is the pair of brackets on line 36 around "name" some new syntax in ES6? I failed to search for its usage